src/Dashboard.tsx

```diff
@@ -32,9 +33,19 @@ function Dashboard() {
       !auth.activeNavigator &&
       !auth.isLoading
     ) {
-      auth.signinSilent();
+      const oidcStorage = sessionStorage.getItem(
+        `oidc.user:${getEnv("VITE_SAMS_AUTHORITY_URL")}:${getEnv(
+          "VITE_SAMS_CLIENT_ID"
+        )}`
+      );
+
+      if (oidcStorage) {
+        auth.signinSilent();
+      } else {
+        navigate("/", { replace: true });
```

Comment:
The errors in my past comments are resolved, but I am still seeing the "Timed out waiting for tcp:127.0.0.1:5173" warning message.  The OIDC lib must be still trying to sign in somehow.  In addition, I am seeing a flash of the dashboard when I click the browser back button after logging out.  We could solve that by moving this `navigate` call outside of the use effect, and using the `Navigate` component instead.

Reply:
Ok, updating now.